build.sbt

```diff
@@ -81,7 +81,17 @@ def slickGeneralSettings =
       "-diagrams", // requires graphviz
       "-groups"
     ),
-    logBuffered := false
+    logBuffered := false,
+    // Adds a `src/main/scala-2.13+` source directory for code shared
```

Comment:
This was taken from https://github.com/apache/incubator-pekko/blob/8e60ef5c6d42c1cd25ad1a3e33bff22e2c9506d1/project/PekkoBuild.scala#L160-L169 and its a well known technique to allow the `scala-2.13+` style source directories which means that when Scala 3 support is merged into main then no additional work is neccessary.